.github/workflows/CI.yml

```
@@ -14,7 +14,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - '1.8'
           - '1'
           - 'nightly'
```

Slightly more stable (and relevant?) would be to test pre-releases:
```suggestion
          - 'pre'
```